public/app/core/components/NavBar/NavBar.tsx

```diff
@@ -51,11 +51,16 @@ export const NavBar = React.memo(() => {
     menuOpen
   );
 
+  let homeUrl = config.appSubUrl || '/';
+  if (!config.bootData.user.isSignedIn && !config.anonymousEnabled) {
+    homeUrl = locationUtil.getUrlForPartial(location, { forceLogin: 'true' });
```

Comment:
when is this ever true? can't the logo always go to `config.appSubUrl || '/'` , what is this scenario for?

Reply:
it's true for snapshots and maybe(?) public dashboards (haven't checked this one yet)

Reply:
Old navigation (before megamenu iirc) used to have this behaviour - we regressed this quite a bit ago.